blackbox/docs/sql/scalar.txt

```diff
@@ -215,6 +215,41 @@ locale-sensitive or context-sensitive mappings.
     SELECT 1 row in set (... sec)
 
 
+sha1('string') returns string
+-----------------------------
+
+Computes the SHA1 checksum of the given string.
+
+::
+
+    cr> select sha1('crate')
+    ... limit 1;
+    +------------------------------------------+
+    | sha1('crate')                            |
+    +------------------------------------------+
+    | 1673dc397042322a0a5ac49c79cc08d3a25cb0f6 |
+    +------------------------------------------+
+    SELECT 1 row in set (... sec)
+
+sha256('string') returns string
```

Comment:
ok, so i propose we then remove also sha256 in order to get this in. there are other crypto functions as well in future and we have not decided on an interface for them, so i would stick with sha1 and md5 since those are known functions at least from mysql and postgresql (md5)

reagding the upstream code ... internal APIs should not lead our decision on user interface.